app/lib/widgets/cards/productCard/shared/price.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flex_ui/utils/extensions.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart' as widgetbook;
+import 'package:intl/intl.dart';
+
+enum Variant {
+  standard,
+  savings, // Defaults to text decoration linethrough and color 'disabled'
+}
+
+class Price extends StatelessWidget {
+  const Price({
+    super.key,
+    required this.price,
+    this.formatterCallback,
```

Comment:
Minor, but can we call this `priceFormatter` ? Hovering it will show the callback details.

Reply:
I flipped around on naming this one far much, priceFormatter is perfect.